examples/f5_bigip/README.md

```diff
@@ -310,7 +298,7 @@ f5_pool_members = [ "192.168.6.201:8001", "192.168.6.201:8002", "192.168.6.201:8
     }
     ```
 
-6. Create you resource in order to create your virtual server to manage your F5 partition ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server)):
+6. Create your resource in order to create your virtual server for managing your F5 partition ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server)):
```

Comment:
Sorry hehe, `aboout`  it's my typo from my PR

Reply:
I'm a writer and I create typos every day (<: 

For the link texts, I suggest you change them to something like "learn more about this configuration here"